astra/src/main/java/com/slack/astra/graphApi/Graph.java

```diff
@@ -0,0 +1,15 @@
+package com.slack.astra.graphApi;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+public class Graph {
+  List<Node> nodes;
+  Set<Edge> edges;
+
+  public Graph(ArrayList<Node> nodes, Set<Edge> edges) {
```

Comment:
nit: this should be
```suggestion
  public Graph(List<Node> nodes, Set<Edge> edges) {
```

But if you do the record thing, then it doesn't matter.